### ISSUE TITLE
Ready to be registered?

### ISSUE THREAD

**Issue by devmotion** [JuliaOptimalTransport/ExactOptimalTransport.jl#3]
In the discussion in https://github.com/ericphanson/UnbalancedOptimalTransport.jl/issues/9 the idea came up that it might be reasonable to split OptimalTransport.jl into separate packages to reduce the number of dependencies (e.g. apparently MOI is a quite heavy dependency) and possibly also precompilation time. Tests pass and it seems the package could be registered.

Are we happy with the name? @zsteve @davibarreira

**Comment by davibarreira** [comment]
Yep. Good for me! Sorry for starting this and not finishing. Hopefully I can go back to coding by the end of the month. Too much stuff right now.

**Comment by zsteve** [comment]
Sounds good! Agree that splitting the packages is a move in the right direction!

**Comment by zsteve** [comment]
P.S. will there be a subsequent PR to OptimalTransport.jl to remove the exact OT functionality?

**Comment by devmotion** [comment]
Yes, I'll make a PR when the package is registered. I just started the registration process, IIRC it should be available in 3 days.
